crates/client/src/client/schema/tx/transparent_tx.rs

```diff
@@ -387,6 +396,43 @@ impl TryFrom<Transaction> for fuel_tx::Transaction {
                     .collect(),
             );
             tx.into()
+        } else if tx.is_blob {
```

Comment:
Do we have any conversion-specific tests for this?

Reply:
No. None of the variants have that kind of tests. You can create an issue descibing what kind of tests you'd like to see for these, if you're fine with leaving this as a follow-up.

Reply:
On second thought, I guess this conversion should be happening as part of some interface. It should just be that interface that is tested, rather than the internal conversion code specifically. And you've added integ tests that test the client which I guess is that interface.

I'll finish my thoughts there.